rpcq/_base.py

```diff
@@ -163,7 +164,10 @@ def from_msgpack(b):
     # Docs for raw parameter are somewhat hard to find so they're copied here:
     #   If true, unpack msgpack raw to Python bytes (default).
     #   Otherwise, unpack to Python str (or unicode on Python 2) by decoding with UTF-8 encoding (recommended).
-    return msgpack.loads(b, object_hook=_object_hook, raw=False)
+    #   In msgpack >= 0.6, max_xxx_len is reduced from 2 GB to 1 MB, so we set the relevant ones
+    #       to 1 GB as to not run into issues with the size of the values returned from rpcq
+    return msgpack.loads(b, object_hook=_object_hook, raw=False,
```

Comment:
Out of curiosity: what sort of error does this throw when the length constraints are violated? (Maybe also add a test?) How is this different from the error that used to happen, when it was left up to the libraries to discover and throw the error? (And why is it different, if the constants we're supplying here are right at the edge of what the libraries consider to be legal?)

Reply:
```
msgpack/_unpacker.pyx in msgpack._cmsgpack.unpackb()

ValueError: 11521764 exceeds max_str_len(10000000)
```

Reply:
this error always could happen, they just lowered the default max in version 0.6 so we actually started hitting it

Reply:
i'll look into making a test now

Reply:
Oh. msgpack would pack it and ZMQ would transmit it, but then msgpack wouldn't unpack it?? Yuck.

Reply:
exactly